test/fixtures/e2e/bento/minimal.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Bento component renders without v0.js</title>
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+    <script
+      async
+      custom-element="amp-timeago"
+      src="https://cdn.ampproject.org/v0/amp-timeago-1.0.js"
+    ></script>
+  </head>
+  <body>
+    <amp-timeago
+      layout="fixed"
+      width="160"
+      height="20"
```

Comment:
Same as above - these `layout`/`width`/`height` can be removed

Reply:
See my comment as well.